.github/workflows/release-comments.yml

```diff
@@ -2,44 +2,22 @@ name: 📝 Comment on Release
 
 on:
   workflow_call:
-    inputs:
-      ref:
-        required: true
-        type: string
-      package_version_to_follow:
-        required: true
-        type: string
-      release_branch:
-        required: true
-        type: string
 
 jobs:
   comment:
-    name: Comment on Release
+    name: 📝 Comment on related issues and pull requests
     if: github.repository == 'remix-run/remix'
     runs-on: ubuntu-latest
     steps:
       - name: ⬇️ Checkout repo
         uses: actions/checkout@v3
-
-      - name: ⎔ Setup node
-        uses: actions/setup-node@v3
         with:
-          node-version-file: ".nvmrc"
-          cache: "npm"
-          cache-dependency-path: scripts/release/package-lock.json
-
-      - name: 📥 Install deps
-        run: npm ci
-        working-directory: ./scripts/release
+          fetch-depth: 0
 
       - name: 📝 Comment on issues
-        working-directory: ./scripts/release
-        run: node -r esbuild-register ./comment.ts
+        uses: mcansh/release-comment-action@v0.1.3
         env:
+          DIRECTORY_TO_CHECK: "./packages"
+          PACKAGE_VERSION_TO_FOLLOW: "remix"
           GITHUB_REPOSITORY: ${{ github.repository }}
-          GITHUB_TOKEN: ${{ github.token }}
-          VERSION: ${{ inputs.ref }}
-          DEFAULT_BRANCH: "main"
-          RELEASE_BRANCH: ${{ inputs.release_branch }}
-          PACKAGE_VERSION_TO_FOLLOW: ${{ inputs.package_version_to_follow }}
+          GH_TOKEN: ${{ github.token }}
```

Comment:
Same comment as https://github.com/remix-run/react-router/pull/10265#discussion_r1148373603

> What about adding this as input instead of ENV variables?
> 
> This way you can set defaults for at least `GITHUB_REPOSITORY` & `GH_TOKEN` & you don't need to pass them anymore as the user of the Action
> You can see an example of having this for `GH_TOKEN` here:
> https://github.com/actions/labeler/blob/7012d51fe062f0b1e26e224a3c9fb52598ee3302/action.yml#L5-L8

Reply:
good suggestion

Reply:
we'll still need GH_TOKEN to be an environment variable, which i guess we could consume as an input and manually set `process.env.GH_TOKEN = core.getInput("GH_TOKEN")`, but seems icky at first glance